examples/policy.yaml

```
@@ -0,0 +1,24 @@
+      - engineering@example.com
+wwwwwh_requirements:
+  prod:
+    require_ticket: true
+    require_service: true
+    require_where: [region, cluster, namespace]
+  default:
+    require_ticket: false
+ff_only: true
+notes_ref: "refs/_shiplog/notes/logs"
+journals_ref_prefix: "refs/_shiplog/journal/"
+anchors_ref_prefix:  "refs/_shiplog/anchors/"
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)